pelita/game.py

```diff
@@ -81,18 +81,101 @@ def controller_exit(state, await_action='play_step'):
         elif todo in ('play_step', 'set_initial'):
             return False
 
-def run_game(team_specs, *, max_rounds, layout_dict, layout_name="", seed=None,
-             max_team_errors=5, timeout_length=3, viewers=None, controller=None, viewer_options=None,
+def run_game(team_specs, *, layout_dict, layout_name="", max_rounds=300, seed=None,
+             max_team_errors=5, timeout_length=3, viewers=None, viewer_options=None,
              store_output=False, team_names=(None, None), allow_exceptions=False):
-    """ Run a match for `max_rounds` rounds. """
+    """ Run a pelita match.
+
+    Parameters
+    ----------
+    team_specs : list[team0, team1]
+              a list specifying the two teams to play the match with. The list
+              items team0 and team1 must be either
+               - a function with signature move(bot, state) -> (x, y), state
+                 the function takes a bot object and a state and returns the next
+                 position (x, y) of bot and state.
+                 state can be an arbitrary Python object, None by default
+               - the path to a Python module that defines at least a function
+                 called 'move' (specified as above), and a string TEAM_NAME.
+
+              When the team_specs are functions a "local" game will be run, when
+              they are paths to Python modules a "remote" game will be run instead.
+              See below in section Notes for more details about local and remote games.
+
+    layout_dict : dict
+               a dictionary representing a maze, as returned by pelita.layout.parse_layout
+    layout_name : str
+               a name for the layout (will be used in the UI).
+
+    max_rounds : int
+              The maximum number of rounds to play before the game is over. Default: 300.
+
+    seed : int
+        seed used to initialize the random number generator.
+
+    max_team_errors : int
+                   The maximum number of non fatal errors for a team before the
+                   game is over and the team is disqualified. Non fatal errors are
+                   timeouts and returning an illegal move. Fatal errors are raising
+                   Exceptions. Default: 5.
+
+    timeout_length : int or float
+                  Time in seconds to wait for the move function (or for the remote
+                  client) to return. After timeout_length seconds are elapsed an
+                  non-fatal error is recorded for the team.
+
+    viewers : list[viewer1, viewer2]
```

Comment:
`list[viewer1, viewer2, ...]` to avoid implying there have to be two viewers (as with the teams)